apps/web/tailwind.config.js

```diff
@@ -71,21 +71,21 @@ module.exports = {
         },
         flip: {
           to: {
-            transform: 'rotate(360deg)'
-          }
+            transform: 'rotate(360deg)',
```

Comment:
Check you eslint / prettier IDE config, maybe it's ignoring the projects settings defaults. It should not format adding this commas, please commit again reseting all the files that are just formatting to make the next reviews easier.

Reply:
in reality, this file is being repaired